searchguard/rolesmapping.py

```diff
@@ -154,8 +167,9 @@ def modify_rolemapping(role, properties, action="replace"):
     _send_api_request(role, properties)
 
 
-def list_rolemappings_for_user(user, roles):
-    """Get list of rolemappings that contain the given user.
+def list_rolemappings_for_user(user, roles=None):
+    """Get list of rolemappings that contain the given user. If no roles are provided,
```

Comment:
Maybe document it like "It is possible to add a list of roles to check. If no list is added, all rolemappings are evaluated"